misc/install-xattr/install-xattr.c

```diff
@@ -352,7 +352,7 @@ main(int argc, char* argv[])
 		case -1:
 			err(1, "fork() failed");
 
-		case 0:
+		case 0: ;
 			char *mypath = realpath("/proc/self/exe", NULL); /* path to argv[0] */
```

Comment:
can't declare a variable in a case statement without another scope?

Reply:
Clang hates it:
```
$ make CC=clang-16 CFLAGS="-std=gnu17 -O2 -pedantic -fsanitize=address -ggdb3" LDFLAGS="-fsanitize=address -ggdb3"
clang-16 -std=gnu17 -O2 -pedantic -fsanitize=address -ggdb3  -fsanitize=address -ggdb3 -o install-xattr install-xattr.c
install-xattr.c:356:4: error: expected expression
                        char *mypath = realpath("/proc/self/exe", NULL); /* path to argv[0] */
                        ^
install-xattr.c:361:20: error: use of undeclared identifier 'mypath'; did you mean 'path'?
                        install = which(mypath, portage_helper_canpath);
                                        ^~~~~~
                                        path
install-xattr.c:249:8: note: 'path' declared here
        char *path;                    /* path to the target file                                      */
              ^
install-xattr.c:362:9: error: use of undeclared identifier 'mypath'; did you mean 'path'?
                        free(mypath);
                             ^~~~~~
                             path
install-xattr.c:249:8: note: 'path' declared here
        char *path;                    /* path to the target file                                      */
              ^
3 errors generated.
make: *** [Makefile:8: install-xattr] Error 1
```

```
$ make CC=gcc CFLAGS="-std=gnu17 -O2 -pedantic -fsanitize=address -ggdb3" LDFLAGS="-fsanitize=address -ggdb3"
gcc -std=gnu17 -O2 -pedantic -fsanitize=address -ggdb3  -fsanitize=address -ggdb3 -o install-xattr install-xattr.c
install-xattr.c: In function ‘main’:
install-xattr.c:356:25: warning: a label can only be part of a statement and a declaration is not a statement [-Wpedantic]
  356 |                         char *mypath = realpath("/proc/self/exe", NULL); /* path to argv[0] */
      |                         ^~~~
```

Reply:
wrap it in a `{}` scope